scripts/nasa/obs/tempo/store_obs.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import os
+
+from r2d2 import R2D2Data
+from solo.configuration import Configuration
+from solo.date import date_sequence, JediDate, DateIncrement
+
+
+def store_observations(yaml_file):
+    config = Configuration(yaml_file)
+    dates = date_sequence(config.start, config.end, config.window_length)
+    provider = config.provider
+    #type = config.type
+    item =  config.item
+    window_length = config.window_length
+    source_path = config.source_path
+    observation_type = config.obs_type
+
+    for date in dates:
+        date = JediDate(date)
+        window_start = date - DateIncrement('PT3H')
+        source_file=f'{source_path}/TEMPO_Intelsat40e_{date.year()}{date.month()}{date.day()}{date.hour()}.nc'
+        if not os.path.isfile(source_file): source_file=f'{source_path}/empty.nc' 
```

Comment:
I know it adds a line, but it's a little hard to keep track at first look. I believe that `pylint` complains about it too.
```suggestion
        if not os.path.isfile(source_file):
            source_file=f'{source_path}/empty.nc' 
```